packages/ubuntu_wsl_setup/lib/pages/profile_setup/profile_setup_widgets.dart

```diff
@@ -55,11 +56,13 @@ class _UsernameFormField extends StatelessWidget {
         PatternValidator(
           kValidUsernamePattern,
           errorText: lang.profileSetupUsernameInvalid,
-        )
+        ),
+        ...model.validators,
```

Comment:
...and then together with an extension for localizing enum values, `...model.validators` could be replaced by a single validator that checks the current validation status from the model?

```dart
CallbackValidator(
  (_) => model.usernameOk,
  errorText: validation.localize(lang),
),
```

Reply:
That suggestion blew my mind! Got where I wanted in a much cleaner way. Thank you very much for the suggestion.